client/my-sites/checklist/main.jsx

```diff
@@ -0,0 +1,171 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+import React, { Fragment, PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import config from 'config';
+import EmptyContent from 'components/empty-content';
+import FormattedHeader from 'components/formatted-header';
+import ChecklistShow from './checklist-show';
+import Main from 'components/main';
+import DocumentHead from 'components/data/document-head';
+import { requestSiteChecklistTaskUpdate } from 'state/checklist/actions';
```

Comment:
Since we're introducing this one, would you like to sort the imports alphabetically here?